src/pages/Recipe.js

```diff
@@ -35,78 +97,226 @@ export default class Recipe extends Component {
       recipe.flavorTotal += flavor.percent;
     });
 
+    if (recipe.notes) {
+      const newNotes = recipe.notes
+        .toString()
+        .split('\n')
+        .map((line, i) => {
+          return <Card.Text key={`${recipe.id}${i}`}>{line}</Card.Text>;
+        });
+
+      recipe.notes = newNotes;
+    }
+
+    const newTags = recipe.tags.map((tag, i) => {
+      return (
+        <a key={i} href="/">
```

Comment:
This guy needs the same fix as above - in general any time you're using `.map()` you need to pass it a key that changes if and only if the actual data being stored changes.